docs/source/traitsui_user_manual/factories_basic.rst

```diff
@@ -503,7 +504,10 @@ If the trait attribute that is being edited is not an enumeration, you must
 specify either the trait attribute (with the *name* parameter), or the set of
 values to display (with the *values* parameter). The *name* parameter can be an
 extended trait name. The *values* parameter can be a list, tuple, or dictionary,
-or a "mapped" trait.
+or a "mapped" trait.  When using *names* or *values*, the *use_separator* flag
+turns the string value specified by the *separator* trait to instead insert a
```

Comment:
something not quite right with the wording here: I don't know how to parse "turns the string value ... to insert a separator"